### PR TITLE
Rayonize compilation and hashing in class-definition-checkpoint-sync

### PR DESCRIPTION
**Pull request by CHr15F0x**
Tested on sepolia in block range ` 220007..=221000`:
- `main`: 212 sec (100%)
- with this PR: 91 sec (42%, ~2.3x speedup)

### CODE REVIEW

File: crates/make-stream/Cargo.toml
[the remark this answers is not in the document]
Comment (CHr15F0x) [reply crates/make-stream/Cargo.toml:1]: Good idea, there's also the AnyhowExt trait that wants to join, badly :D.

[the remark this answers is not in the document]
Comment (CHr15F0x) [reply crates/make-stream/Cargo.toml:1]: In #2299

**Review by t00ts — APPROVED** [review]
Clean 👌

**Review by kkovaacs — APPROVED** [review]

**Comment by CHr15F0x** [comment]
Ad discussed offline: increased chunk sizes, reverted concurrent layout verification.

**Pull request merged**